src/app/app-config.service.ts

```diff
@@ -9,6 +9,11 @@ export interface Config {
     background: string;
   };
   logo: string;
+  about: {
+    title: string,
+    body: string,
+    buttons: [{title: string, url: string}]
```

Comment:
Shouldn't this be
```suggestion
    buttons: {title: string, url: string}[]
```

i.e. array of T instead of tuple with one entry of type T